pkg/receive/handler.go

```diff
@@ -100,12 +100,18 @@ func NewHandler(logger log.Logger, o *Options) *Handler {
 		router:  route.New(),
 		options: o,
 		peers:   newPeerGroup(o.DialOpts...),
-		forwardRequestsTotal: promauto.With(o.Registry).NewCounterVec(
+		forwardRequests: promauto.With(o.Registry).NewCounterVec(
 			prometheus.CounterOpts{
 				Name: "thanos_receive_forward_requests_total",
 				Help: "The number of forward requests.",
 			}, []string{"result"},
 		),
+		replications: promauto.With(o.Registry).NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "thanos_receive_replications_total",
+				Help: "The number of replication operation.",
```

Comment:
```suggestion
				Help: "The number of replication operations.",
```

Reply:
I would also add more info like
```suggestion
				Help: "The number of replication operations started by the receiver. The success of replication is fulfilled when a quorum is met",
```